### ISSUE TITLE
Goto non-existing location after loading sequence dictionary

### ISSUE THREAD

**Issue by dariober** [dariober/ASCIIGenome#26]
The following steps lead to an uncaught exception:
- Load a file without sequence dictionary. E.g.

```
ASCIIGenome hg19.gencode_genes_v19.gtf.gz
```
- Go to a non existing location (allowed since there is no sequence dictionary)

```
goto FOOBAR
```

1.  If now you load a file with dictionary like a bam file you get:

```
[h] for help: addTracks ear045.oxBS.actb.bam

Error processing input: [ear045.oxBS.actb.bam]

[h] for help: 
```

That's ok-ish, error could be more informative though.

2. If you go to an existing location, load the bam file and move to _previous_ position:

```
goto chr1 && addTracks ear045.oxBS.actb.bam && p

Exception in thread "main" java.lang.reflect.InvocationTargetException
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:58)
Caused by: java.lang.NullPointerException
    at samTextViewer.GenomicCoords.getChromIdeogram(GenomicCoords.java:396)
    at samTextViewer.TrackProcessor.iterateTracks(TrackProcessor.java:41)
    at samTextViewer.Main.main(Main.java:141)
    ... 5 more
```

This is no good, although going to non existing locations repeatedly is something you don't normally do.

**Comment by dariober** [comment]
Now ASCIIGenome doesn't throw the horrible stack trace and it doesn't exit. 

Point 1. is fixed with issue#23 by (silently) moving to an existing location.

Point 2. Gives `Error processing tracks with input [p]`, not great but at least ASCIIGenome doesn't terminate.

**Comment by dariober** [comment]
Still some issues - version 0.6.3:

    ASCIIGenome
    goto chr1
    setGenome mm9
    p

```
Initializing coordinates... 
Cannot find chromosome 'Undefined_contig' in sequence dictionary.
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.eclipse.jdt.internal.jarinjarloader.JarRsrcLoader.main(JarRsrcLoader.java:58)
Caused by: exceptions.InvalidGenomicCoordsException
	at samTextViewer.GenomicCoords.<init>(GenomicCoords.java:81)
	at samTextViewer.InteractiveInput.processInput(InteractiveInput.java:366)
	at samTextViewer.Main.main(Main.java:174)
	... 5 more

```

**Comment by dariober** [comment]
Fixed in v0.6.4. Invalid genomic positions are removed from history after having set genome.
